project-reports/2023/2023-Q2-Hyperledger-Besu.md

```diff
@@ -0,0 +1,55 @@
+---
+layout: default
+title: Q2 Hyperledger Besu
+parent: 2023
+grand_parent: Project Updates
+---
+
+# Project Health
+
+Hyperledger Besu is gaining more and more traction across public and private Ethereum and Ethereum-like networks. There is an uptick in contributions on the private network side with new contributors from Web3Labs and Kaleido moving towards maintainer status. The Besu strategy in the coming quarters is looking to be driven by more than just Consensys, with a focus on tech debt resolution that will simplify maintaining a Swiss-army knife client across many network types. These are also means for the current maintainers to engage new developers on the code-base and improve contributor skill, quality, and number. Discord activity remains heavy, with a relatively even split between public and private network questions and suggestions. The project is seeing some key contributors step away to focus on other areas, but we hope to fill the gap with new ones and leverage some of their knowledge in upskilling.
+
+# Questions/Issues for the TOC
+
+Have we been able to complete the asks around CircleCI funding by Hyperledger Foundation?
+
+# Releases
+
+There were only three Besu releases this quarter, two point releases with fixes and a huge quarterly with a number of new changes. We expect another rather large quarterly in July with a huge amount of features focused on performance, fixes, and tech debt. We are also using these quarterlies to deprecate older features relating to Quorum, which is no longer being maintained by Consensys. 
```

Comment:
great to hear about the significant releases. might be a good idea if you can add a bit details on "older features relating to Quorum". Thanks!

Reply:
We removed compatibility with GoQuorum privacy in 23.4.0 and are removing compatibility with GoQuorum-compatible permissioning in 23.7.0.